Source/Core/InputCommon/ControllerInterface/WGInput/WGInput.cpp

```diff
@@ -0,0 +1,536 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <algorithm>
+#include <array>
+
+#include <roapi.h>
+
+#include "Common/CommonFuncs.h"
+#include "Common/Logging/Log.h"
+#include "Common/StringUtil.h"
+#include "InputCommon/ControllerInterface/WGInput/WGInput.h"
+
+#pragma comment(lib, "runtimeobject.lib")
+
+namespace ciface
```

Comment:
These can be collapsed into

```cpp
namespace ciface::WGInput
{
```

since we use C++17

The static locals can also be enclosed in an anonymous namespace.